scripts/linear-ticket/validate.py

```
@@ -0,0 +1,459 @@
+    def __init__(self, repo: str):
+        self.repo = repo
+
+    def _run(self, args: list[str], stdin: str | None = None) -> subprocess.CompletedProcess:
+        return subprocess.run(
+            ["gh", "api", *args],
+            input=stdin,
+            text=True,
+            capture_output=True,
+        )
+
+    def get(self, path: str, paginate: bool = False):
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)